app/gui2/src/composables/navigator.ts

```diff
@@ -99,7 +99,8 @@ export function useNavigator(
     maxScale = PAN_AND_ZOOM_DEFAULT_SCALE_RANGE[1],
     skipAnimation = false,
   ) {
-    if (!viewportNode.value) return
+    if (!viewportNode.value || !viewportNode.value.clientHeight || !viewportNode.value.clientWidth)
```

Comment:
Why is this check necessary now? I'm worried that a pan/zoom action at load can now be silently skipped. If for some reason this is called when viewportNode doesn't have. size yet, then we have to fix the root cause of that. Maybe this should be an assertion instead of early return.

Reply:
oh. right, now that i think about it... the cause is that viewportNode has size = 0 when the editor isn't visible. what should we do? cache the last known size? or perhaps just delay reloading this stuff until viewportnode is visible again?

Reply:
Delay sounds good in this case.